### ISSUE TITLE
Flatten the WooCommerce menu in Ecommerce plan

### ISSUE THREAD

**Issue by xristos3490** [Automattic/wc-calypso-bridge#850]
### Description

The objective of this work stream is to investigate the feasibility of flattening the WooCommerce menu item as follows:

- Move the **Orders** menu item to the top level of our navigation, and expose All Orders and New Order submenu items.
- Move the **Customers** menu item to the top level of our navigation.
- Move **WooCommerce > Status** to **Tools > WooCommerce Status**.
- Move **WooCommerce > Settings** to **Settings > WooCommerce**.
- Drop the legacy, now-deprecated **WooCommerce > Reports** section.
- Finally, relabel the **WooCommerce** menu item to **Extensions** and move **WooCommerce > Extensions** to the top as **Extensions > Discover**.

In a nutshell, the plan is to pull our submenu items out of the WooCommerce menu, and leave this space for use by Extensions — which already add their menu items there.

More at peapX7-Um-p2
